### PR TITLE
Removed `...props` from SubmitField section

### PR DESCRIPTION
**Pull request by zeroasterisk**
> Warning: Unknown prop `grid` on `section` tag.
> Remove this prop from the element

### CODE REVIEW

**Comment by radekmie** [comment]
Since #64 (see [this](https://github.com/vazco/uniforms/issues/64#issuecomment-233732854) comment), there's a `filterDOMProps` helper. [Example](https://github.com/vazco/uniforms/blob/master/packages/uniforms-semantic/src/components/fields/BoolField.js#L19).

---

I'll add it to README later.

**Comment by zeroasterisk** [comment]
updated, thanks!

File: packages/uniforms-bootstrap3/src/components/fields/ErrorsField.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import {nothing}   from 'uniforms';
 
 const ErrorsField = ({className, children, ...props}, {uniforms: {error, schema}}) =>
     (!error && !children) ? nothing : (
-        <section className={classnames('panel panel-danger', className)} {...props}>
+        <section className={classnames('panel panel-danger', className)} {...filterDOMProps(props)}>
```
Comment (radekmie) [inline packages/uniforms-bootstrap3/src/components/fields/ErrorsField.js]: You haven't imported this helper - ESlint fails.

**Comment by radekmie** [comment]
I've continued your PR and done it also in `semantic` and `unstyled` packages. Thanks for pointing it out!

**Pull request closed without merging**